tests/sparseml/transformers/sparsification/modification/test_modifying_mistral.py

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from copy import deepcopy
+
+import pytest
+from transformers import AutoConfig, AutoModel
+
+from accelerate import init_empty_weights
+from sparseml.transformers.sparsification.modification import modify_model
+
+
+@pytest.fixture
+def mistral_model():
+    config = AutoConfig.from_pretrained("mistralai/Mistral-7B-v0.1")
+    with init_empty_weights():
+        model = AutoModel.from_config(config)
+    return model
```

Comment:
nice! so not much memory gets allocated but we get to try a "real size" model?

Reply:
did not investigate the memory requirement, but this model gets initialized in a fraction of a second :)